appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -680,11 +680,12 @@ private YaVersion() {
 
   public static final int GYROSCOPESENSOR_COMPONENT_VERSION = 1;
 
-  // For HORIZONTALARRANGEMENT_COMPONENT_VERSION 2:
+  // For HORIZONTALARRANGEMENT_COMPONENT_VERSION 3:
+  // - The scrollable property was added
   // - The AlignHorizontal property was added
   // - The AlignVertical property was added
```

Comment:
Before the line "// - Added background color & image" we need to add:
    // For HORIZONTALARRANGEMENT_COMPONENT_VERSION 3:
This is to fix a bug from a previous change (when we added the background color and image, this line was left out.
The similar change needs to be added for VERITICALARRANGEMENT_COMPONENT_VERSION as well.

This is *not* your issue, but given you are updating this, it is a good time to correct the problem. Thanks.

-Jeff